content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -241,12 +228,47 @@ getting replaced.
 
 Due to the nature of EndpointSlice changes, endpoints may be represented in more
 than one EndpointSlice at the same time. This naturally occurs as changes to
-different EndpointSlice objects can arrive at the Kubernetes client watch/cache
-at different times. Implementations using EndpointSlice must be able to have the
-endpoint appear in more than one slice. A reference implementation of how to
-perform endpoint deduplication can be found in the `EndpointSliceCache`
-implementation in `kube-proxy`.
+different EndpointSlice objects can arrive at the Kubernetes client watch / cache
+at different times.
+
+{{< note >}}
+Clients of the EndpointSlice API must be able to handle the situation where
+a particular endpoint address appears in more than one slice.
+
+You can find a reference implementation for how to perform this endpoint deduplication
+as part of the `EndpointSliceCache` code within `kube-proxy`.
```

Comment:
this was suggested by Rob or someone from sig-network?

I personally don't feel that the kube-proxy cache is a good example, there are more simpler ways of implementing this

Reply:
This PR is mostly a refactor - see https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/#duplicate-endpoints for the original text.